src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const BASE_PRICE = 40;
+
+  if (days < SHORT_TERM) {
+    return days * BASE_PRICE;
+  }
+
+  if (days < LONG_TERM) {
+    return (days * BASE_PRICE) - SHORT_TERM_DISCOUNT;
+  }
+
+  return (days * BASE_PRICE) - LONG_TERM_DISCOUNT;
```

Comment:
like this will be more understanded
```suggestion

  if (days > LONG_TERM) {
      return (days * BASE_PRICE) - LONG_TERM_DISCOUNT;
  }

  if (days > SHORT_TERM) {
      return (days * BASE_PRICE) - SHORT_TERM_DISCOUNT;
  }

  return days * BASE_PRICE;

```